packager/media/codecs/nal_unit_to_byte_stream_converter.cc

```diff
@@ -337,6 +337,8 @@ bool NalUnitToByteStreamConverter::ConvertUnitToByteStreamWithSubsamples(
           break;
         FALLTHROUGH_INTENDED;
       }
+      case Nalu::H264_SPSExtension:
+        FALLTHROUGH_INTENDED;
```

Comment:
That is incorrect. Please move it after line 318. We want to fall through to the code executed in H264_PPS.

Can you also update nal_unit_to_byte_stream_converter_unittest.cc?